core/sequencer/codes/code.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codes
+
+import "google.golang.org/grpc/codes"
+
+// Code is an enum that represents a kind of error.
+type Code int
+
+const (
```

Comment:
You don't need these to actually be error codes. Using an enum or fixed string would work just as well. This seems like it will be setting us up for needing to maintain an error code for every possible error which I don't believe is desirable.